honeycomb/core/src/main/java/org/commonjava/o11yphant/honeycomb/impl/GoldenSignalsRootSpanFields.java

```diff
@@ -69,15 +67,20 @@ public GoldenSignalsRootSpanFields( GoldenSignalsMetricSet goldenSignalsMetricSe
         //
         // Or, we can use a GROUP BY traffic_type and see the different traffic types broken down in a single graph.
         //
-        final Map<String, Metric> metrics = goldenSignalsMetricSet.getMetrics();
+        Collection<String> functions = RequestContextHelper.getContext( GOLDEN_SIGNALS_FUNCTIONS );
+        if ( functions == null || functions.isEmpty() )
+        {
+            return ret;
+        }
+
         Set<String> classifierTokens = new LinkedHashSet<>();
```

Comment:
Does classifierTokens need to be sorted in favor of querying?

Reply:
The query just uses `contains` so I don't think sorting would help much.